changelogs/unreleased/anomalies-on-new-project-features-on-6.2.yml

```diff
@@ -0,0 +1,3 @@
+---
+title: "Project : anomalies on new project features on 6.2"
```

Comment:
Please remove this changelog entry, as we already have one for project changes in 6.2